src/blockchain/Blockchain.ts

```diff
@@ -47,56 +52,105 @@ export interface IBlockchainApi {
     identity: Identity,
     tx: SubmittableExtrinsic
   ): Promise<SubmittableExtrinsic>
+  submitSignedTx(
+    tx: SubmittableExtrinsic,
+    identity?: Identity,
+    opts?: SubscriptionPromiseOptions
+  ): Promise<SubmittableResult>
   submitTx(
     identity: Identity,
     tx: SubmittableExtrinsic,
     opts?: SubscriptionPromiseOptions
   ): Promise<SubmittableResult>
-  getNonce(accountAddress: string): Promise<Codec>
+  getNonce(accountAddress: string): Promise<BN>
+  reSignTx(
+    identity: Identity,
+    tx: SubmittableExtrinsic
+  ): Promise<SubmittableExtrinsic>
 }
+const TxOutdated = '1010: Invalid Transaction: Transaction is outdated'
+const TxPriority = '1014: Priority is too low:'
+const TxAlreadyImported = 'Transaction Already'
```

Comment:
we need good integration tests for these

Reply:
Please make a ticket and provide information and ideas on how to trigger the errors

Reply:
Created Ticket.

Reply:
https://github.com/KILTprotocol/ticket/issues/845